File: tomchaccom/프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    
    for c in commands:
        i = c[0] -1
        j = c[1]
        k = c[2] -1
        
        ar = array[i : j]
        arr = sorted(ar)
        answer.append(arr[k])
        
    return answer

File: yetpi0413/프로그래머스/1/42748. K번째수/K번째수.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for(auto v : commands){
        int i=v[0];
        int j=v[1];
        int k=v[2];
        vector<int> tmp = vector<int>(array.begin()+(i-1), array.begin()+j);
        sort(tmp.begin(), tmp.end());
        answer.push_back(tmp[k-1]);
    }
    return answer;
}